### DISCUSSION TITLE
GSConnect installation archive location

### DISCUSSION THREAD

**Discussion by smbdar** [andyholmes/gnome-shell-extension-gsconnect#1487] [Q&A] [answered]
Hello,

wiki "https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Installation#install-from-zip" instructs to use "gsconnect@andyholmes.github.io.zip" to install extension and to go to "Latest release" page to download it. On the page there are two assets but they consist source code, not mentioned archive.

Where is "gsconnect@andyholmes.github.io.zip"?

**Comment by andyholmes** [comment, marked as answer]
Sorry about that. It looks like a few releases are missing the upload on GitHub.

[v54](https://github.com/GSConnect/gnome-shell-extension-gsconnect/releases/tag/v54) has one of these uploads, or there's a nightly available for each commit: https://nightly.link/GSConnect/gnome-shell-extension-gsconnect/workflows/main/main/test-build.zip
